src/components/Markdown/Text.tsx

```diff
@@ -40,24 +40,28 @@ export const A: AnchorComponent = ({ children, href, ...linkProps }) => {
 export const Blockquote: FunctionComponent = ({ children }) => (
   <Box
     as="blockquote"
-    bg="gray.100"
     borderLeft="5px solid"
-    borderLeftColor="gray.200"
-    borderRadius="md"
-    p={2}
+    borderLeftColor="gray.100"
+    marginTop="2em"
+    marginX="0.5em"
+    px="1em"
     wordBreak="break-word"
   >
     {children}
   </Box>
 );
 
 export const Em: FunctionComponent = ({ children }) => (
-  <Box as="em" color="blue.500" fontStyle="normal" fontWeight="semibold">
+  <Box as="em" fontStyle="normal" fontWeight="bold">
     {children}
   </Box>
 );
 
-export const P: FunctionComponent = ({ children }) => <Text>{children}</Text>;
+export const P: FunctionComponent = ({ children }) => (
+  <Text lineHeight="1.5" marginBottom="1em" marginTop="1em" marginX="0.5em">
```

Comment:
nitpick, but this could use shorthands to avoid redundant margin definitions

`<Text lineHeight="1.5" my={4} mx={2}>`